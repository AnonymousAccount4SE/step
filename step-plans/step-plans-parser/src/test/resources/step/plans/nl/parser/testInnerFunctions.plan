Set var = "hallo"

Function name="test2"
    test3 vorname = "Franz" Name = "${input.getString('Name')}"

    Function name="test3"
        Echo "$var ${input.getString('vorname')} ${input.getString('Name')}"
    EndFunction
EndFunction

Function name="test1"
    test2 Name = "${input.getString('Name')}"
EndFunction

test1 Name="Muster"