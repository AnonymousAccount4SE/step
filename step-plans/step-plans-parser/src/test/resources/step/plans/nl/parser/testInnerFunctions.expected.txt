TestCase:PASSED:
 Session:PASSED:
  Set:PASSED:
  CallKeyword:PASSED:
   Sequence:PASSED:
    CallKeyword:PASSED:
     Sequence:PASSED:
      CallKeyword:PASSED:
       Sequence:PASSED:
        Echo:PASSED:
